settings.py

```diff
@@ -1,2 +1,11 @@
-client_id = 'Your client_id from https://stepic.org/oauth2/applications/'
-client_secret = 'Your client_secret from https://stepic.org/oauth2/applications/'
+"""
+client_id - 'Your client_id from https://stepic.org/oauth2/applications/'
+client_secret - 'Your client_secret from https://stepic.org/oauth2/applications/'
+"""
+
+import os
+
+client_id = os.environ.get('STEPIC_CLIENT_ID')
+client_secret = os.environ.get('STEPIC_CLIENT_SECRET')
```

Comment:
For me, it seems strange to keep keys for one script in Environment variables. Of cause, it can being set before script execution, instead of keeping it in ./bashrc or any other, but still, I don't get idea why it's useful.